darts/timeseries.py

```diff
@@ -710,6 +710,9 @@ def from_dataframe(
         if not time_index.name:
             time_index.name = time_col if time_col else DIMS[0]
 
+        if series_df.columns.name:
+            series_df.columns.name = None
```

Comment:
forgot it before, but we need to make sure that we don't mutate the original `df`